gt/main.c

```diff
@@ -1164,8 +1408,12 @@ alloc_gt_conf(void)
 }
 
 static inline void
-cleanup_gt_instance(struct gt_instance *instance)
+cleanup_gt_instance(struct gt_config *gt_conf, struct gt_instance *instance)
 {
+	flush_notify_pkts(gt_conf, instance);
```

Comment:
`flush_notify_pkts()` may be called with `instance->ggu_pkts == NULL` here. This would produce a segmentation fault. See label `cleanup: ` of `config_gt_instance()` to see how this condition can be triggered.